Tasks/split.js

```diff
@@ -1,10 +1,12 @@
 // Split an array into two parts
+'use strict';
 
-let split_Array = (index, array) => {
-  begin = array.slice(0, index);
-  const len = array.length;
-  array = array.slice(index, len);
-  return [begin, array];
+const splitArray = (divider, array) => {
+  const START = 0;
+  const firstArray = array.slice(START, divider);
+  const end = array.length;
+  const secondArray = array.slice(divider, end);
+  return [firstArray, secondArray];
 };
 
-require('../Tests/split.js')(split_Array);
+require('../Tests/split.js')(splitArray);
```

Comment:
Also, it's a good practice to end your code with an empty newline (good for working with vcs)